### ISSUE TITLE
Error in menu

### ISSUE THREAD

**Issue by PawelGPL** [microsoft/terminal#9204]
I noticed a small error on the menu: in item "Settings" there is "Ctrl Alt ,"  shortcut instead of "Ctrl ," (open _settings.json_).

# Environment

Microsoft Windows Version 10.0.19042.804
# Steps to reproduce

# Expected behavior

# Actual behavior

**Comment by zadjii-msft** [comment]
Can you share your `settings.json`? Also, what keyboard layout are you using?

**Comment by PawelGPL** [comment]
Default _settings.json_. It does not matter. Even after deleting this file there is the mentioned error in the menu. Clicking on a menu item works like shortcut "Ctrl ," not like  "Ctrl Alt ,".
![obraz](https://user-images.githubusercontent.com/52577731/108361586-f9172500-71f2-11eb-92c8-bc90a64b3204.png)

**Comment by zadjii-msft** [comment]
@PawelGPL What keyboard layout are you using? Do you need to press the <kbd>AltGr</kbd> key to type the `,` character?

**Comment by PawelGPL** [comment]
I don't need to use Alt not AltGrto type , (comma).

**Comment by zadjii-msft** [comment]
Hmm. Something weird is going on here. 

#8967 reminded me that we actually do bind that keybinding by default:

```json
  { "command": "openSettings", "keys": "ctrl+," },
  { "command": { "action": "openSettings", "target": "defaultsFile" }, "keys": "ctrl+alt+," },
```

But not to open the settings file - it's bound to opening the defaults file. 

* What version of the terminal are you using?
* What keyboard layout are you using?

**Comment by PawelGPL** [comment]
I have current version - 1.53.2
I have exactly the same setting in _default.json_
I'm using US keyboard with specific Polish characters obtained by pressing with AltGr, e.g AltGr-c -> ć.
All works ok. except wrong description in menu:
Ctrl+, opens _settings.json_ just like clicking in _Settings_ in menu
Ctrl+Atl+, opens  _defaults.json_

**Comment by zadjii-msft** [comment]
Son of a gun they'll do it. If you paste that action:

```json
  { "command": { "action": "openSettings", "target": "defaultsFile" }, "keys": "ctrl+alt+," },
```

into the actions on 1.5, it'll use that one as the help shortcut to use, not the actual settings action. I think that's already fixed on 1.6, but I don't know when or where we fixed that. We might have a dupe somewhere...

**Comment by PawelGPL** [comment]
Btw. is it possible to install terminal without a MS Store? E.g. in Windows 8.1? Is there a standalone installer with an executable file?

**Comment by DHowett** [comment]
> Btw. is it possible to install terminal without a MS Store? E.g. in Windows 8.1? Is there a standalone installer with an executable file?

No, sorry.

**Comment by zadjii-msft** [comment]
Alright, so looking through the history, I think this was broken in (the combo of #7802 and #8725) and was fixed in #8670. That PR didn't have an original issue tracking it, but rest assured, this is already fixed ☺️
